server/database/queries/index.js

```diff
@@ -1,9 +1,11 @@
 const getAllProducts = require('./getAllProducts');
 const getClients = require('./getClients');
+const getProductById = require('./getProductById');
 const getOrders = require('./getOrders');
```

Comment:
please arrange all related routes by their category
```suggestion
const getAllProducts = require('./getAllProducts');
const getProductById = require('./getProductById');
const getClients = require('./getClients');
const getOrders = require('./getOrders');
```